content/Assets/Scripts/index.js

```diff
@@ -2,6 +2,5 @@
  * Import main Sass file so it is compiled by webpack in to
  * CSS file that can be referenced within `Views/Layout.cshtml`.
  */
-import '../Styles/index.scss';
```

Comment:
@mobinzk If we're removing the import, then let's remove the comment above to avoid any confusion.